src/components/Slide/player.css

```
@@ -253,13 +376,33 @@
+  --slide-player-bottom-offset: 0px;
+  transform: none;
+  left: 12px;
+  right: 12px;
+  width: auto;
+}
+
+.slide--mobile-device.slide--mobile-landscape-native
   .slide-interaction-overlay {
-    --slide-player-bottom-offset: 0px;
-  }
+}
```

Stylelint (17.6.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)